draft-ietf-teep-architecture.md

```diff
@@ -350,9 +350,9 @@ all components are further explained in the following paragraphs.
 
     The TAM performs its management of TA's through an
     interaction with a Device's TEEP Broker. As shown in
-    {{notionalarch}}, the TAM cannot directly contact a Device, but must
+    {{notionalarch}}, the TAM cannot directly contact a TEEP Agent, but must
     wait for the TEEP Broker to contact
-    the TAM requesting a particular service. This architecture is
+    the TEEP Agent requesting a particular service. This architecture is
```

Comment:
I believe the fix in 353 is correct, but the change in 355 is incorrect.   It's talking about the TAM waiting, so it really is talking about the TEEP Broker contacting the TAM.

Reply:
Made the update based on your comment